content/docs/user-guide/basic-concepts.md

```diff
@@ -0,0 +1,34 @@
+# Basic Concepts of DVC
+
+DVC streamlines large data files and binary models into a single Git
+environment. This approach will not require storing binary files in your Git
+repository.
+
+![](/img/flow-large.png) _DVC data management_
+
+### Data Files
+
+Large files (or directories) that are tracked and cached by DVC. Data files are
+stored outside of the Git repository, on a local/shared hard drive, and/or
+remote storage. `.dvc` files describing the data are put into Git as
+placeholders, for DVC needs (to maintain pipelines and reproducibility).
+
+### Local Cache
+
+Directory with all data files on a local hard drive or in cloud storage, but not
```

Comment:
local ... cloud ... sound weird

aslo Local Cache does not sound like a basic concept

Reply:
Agree. For #72